editoast/src/core/v2/path_properties.rs

```diff
@@ -0,0 +1,84 @@
+use editoast_common::geometry::GeoJsonLineString;
+use editoast_common::Identifier;
+use editoast_schemas::infra::OperationalPointExtensions;
+use editoast_schemas::infra::OperationalPointPart;
+use serde::Deserialize;
+use serde::Serialize;
+use utoipa::ToSchema;
+
+use crate::core::{AsCoreRequest, Json};
+use crate::views::v2::path::TrackRange;
```

Comment:
Not blocking or anything but I'd like to avoid having `editoast::core` depend on `views` (that's extra work for the split in crates ; one struct is fine though :))

Reply:
Good catch